### DISCUSSION TITLE
Welcome to bootstraped-multi-test-results-report-plugin Discussions!

### DISCUSSION THREAD

**Discussion by BogdanLivadariu** [jenkinsci/bootstraped-multi-test-results-report-plugin#123] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BogdanLivadariu** [comment]
@NotMyFault  do you think you can help integrate the automated deployment of the pacakges&hpi to https://repo.jenkins-ci.org ?

**Reply by NotMyFault** [reply]
For sure, the process is documented at https://www.jenkins.io/doc/developer/publishing/releasing-cd/
